apps/web/src/components/Home/Trending.tsx

```diff
@@ -22,10 +23,18 @@ const Title = () => {
 };
 
 const Trending: FC = () => {
+  const [withTag, setWithTag] = useState<TagResult[]>([]);
   const { data, loading, error } = useTrendingQuery({
     variables: { request: { limit: 7, sort: TagSortCriteria.MostPopular } }
   });
 
+  useEffect(() => {
+    const t = data?.allPublicationsTags?.items.filter((i) => i?.tag !== '{}');
+    if (t) {
+      setWithTag(t);
+    }
+  }, [data]);
```

Comment:
Do we really need useEffect for this? IMO it should be used to call only for external app calls like APIs and tracking

https://react.dev/learn/you-might-not-need-an-effect